test/Psr7ResponseTest.php

```diff
@@ -78,6 +78,8 @@ public function testResponseToZendFromRealStream($body, $status, $headers)
                 $this->assertContains($value, $zendHeaders[$type]);
             }
         }
+
+        unlink(__DIR__ . '/../test.file');
```

Comment:
See my previous comment. If you use a temporary file generated on the fly you can even omit this. If you use a fixed file in `test/TestAsset` you need to omit this unlink.